technologies.js

```diff
@@ -35,6 +38,16 @@ for (let filename of files) {
     if (!Array.isArray(lib.tags) || lib.tags.length == 0) {
       throw new Error(`${filename}: no tags for ${lib.id}`);
     }
+
+    if (ids.has(lib.id.toLowerCase())) {
```

Comment:
If i was doing it, I would do toLowerCase just once:
```
lib = lib.id.toLowerCase()
repo = repo.id.toLowerCase()
```

Not terribly important.

Reply:
k